cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -63,6 +63,20 @@ func CLITestSuite(
 			testSuiteName, fmt.Sprintf("[%v][CLI] %v", testType, "Import with different network param styles"))
 		imageImportWithSubnetWithoutNetworkSpecifiedTestCase := junitxml.NewTestCase(
 			testSuiteName, fmt.Sprintf("[%v][CLI] %v", testType, "Import with subnet but without network"))
+		imageImportLinuxUEFITestCase := junitxml.NewTestCase(
+			testSuiteName, fmt.Sprintf("[%v][CLI] %v", testType, "Import UEFI test for linux UEFI"))
+		imageImportLinuxUEFIFromImageTestCase := junitxml.NewTestCase(
+			testSuiteName, fmt.Sprintf("[%v][CLI] %v", testType, "Import UEFI from image test for linux UEFI"))
```

Comment:
I *think* this is a good opportunity for this suite instead. Just copy the two UEFI tests, and replace the uefi_compatible arg with `inspect`, right?

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools_e2e_test/gce_image_import_export/test_suites/import/import_tests.go#L70-L69

Reply:
Great! I moved all the new e2e tests to import_tests